osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -99,16 +99,18 @@ protected virtual Triangle CreateTriangle()
 
             const float size = 100;
 
-            return new Triangle
+            return new EquilateralTriangle
             {
                 Origin = Anchor.TopCentre,
+

```

Comment:
that's fine and all, but we're also going for a consistent coding style (and this is inconsistent at the moment). maybe put this up for discussion and we can do a pass on the whole codebase, but don't use this for now?